src/model_output_summary.R

```diff
@@ -0,0 +1,60 @@
+model_output_summary <- function(target_date){
+  ## A function to write a summary table of `model-output` and `model-metadata`
+  ## to PDF format. 
+  # Input: target_date = a model submission date in date format. 
+  
+  library(arrow)
+  library(yaml)
+  library(knitr)
+  library(kableExtra)
+  library(grid)
+  
+  if(!(any(target_date == seq(as.Date("2024-10-09"), Sys.Date(), by = 7)))){
+    stop("The target date is not on a valid Wednesday")
+  }
+  
+  file_directory_metadata <- "./model-metadata/"
```

Comment:
I suggest making lines 16 and 17 arguments for the function with the values specified here as the defaults.